src/components/Messages/extensions/Channel.js

```diff
@@ -89,9 +93,15 @@ const Channel = ({ user = null, db = null }) => {
               type="text"
               value={newMessage}
               onChange={handleOnChange}
+              id="myInput"
+              // onfocus="this.value=''" This does not work either
               placeholder="Type you message here..."
             />
-            <button type="submit" disabled={!newMessage}>
+            <button
+              type="submit"
+              disabled={!newMessage}
+              // onclick="document.getElementById('myInput').value = ''"
```

Comment:
can simply update the state to `' '` to clear the text-area

Reply:
const [messages, setMessages] = useState([]);
const [newMessage, setNewMessage] = useState(""); 

do you mean these?

Reply:
line 103: onclick="document.getElementById('myInput').e.target.value= ''"

None of this is working but the codespace keeps crashing and I'm getting frustrated. I'm taking a brake.